src/tests/Common/XUnitLogChecker/XUnitLogChecker.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <XUnitLogCheckerLibrariesHelixDirectory>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'helix', 'tests', 'XUnitLogChecker'))</XUnitLogCheckerLibrariesHelixDirectory>
```

Comment:
This property is used by both XUnitLogChecker and sendtohelixhelp and should therefore be centrally defined.

Reply:
One's for the build output, another one is for the zipping output. I realized the helix one needed to be located under helix/tests to ensure the zip was uploaded.